packages/at_client/lib/src/encryption_service/abstract_atkey_encryption.dart

```diff
@@ -15,16 +15,16 @@ import 'package:at_chops/at_chops.dart';
 abstract class AbstractAtKeyEncryption implements AtKeyEncryption {
   late final AtSignLogger _logger;
   late String _sharedKey;
-  final AtClient _atClient;
+  final AtClient atClient;
```

Comment:
why has the visibility been increased from _atClient to atClient?

Reply:
in LocalKeyDecryption i had call to super.atClient. 
Modified the code in LocalKeyDecryption to remove call to super.atClient